### PR TITLE
Added zoom to mouse preference

### PR DESCRIPTION
**Pull request by nallath**
https://github.com/Ultimaker/Cura/issues/1572

See also; https://github.com/Ultimaker/Uranium/pull/264

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Devs, see issue CURA-3587.

File: resources/qml/Preferences/GeneralPage.qml [hunk 1]
```diff
@@ -355,6 +357,20 @@ UM.PreferencesPage
...
 
             UM.TooltipArea {
+                width: childrenRect.width;
+                height: childrenRect.height;
+                text: catalog.i18nc("@info:tooltip", "Should zooming move in the direction of the mouse?")
+
+                CheckBox
+                {
+                    id: zoomToMouseCheckbox
+                    text: catalog.i18nc("@action:button","Zoom toward mouse direction");
```
Comment (Ghostkeeper) [inline resources/qml/Preferences/GeneralPage.qml]: Code style: Space after comma. I've fixed that for you here: ef89fab1269cc595fda15ea03d9760789efc3d29

**Comment by fieldOfView** [comment]
Yay!

Note: zooming with +/- still zooms to the center of the window.

**Comment by nallath** [comment]
I wasn't sure what to do in that regard. It also zooms in direction of the mouse when zooming out.

**Comment by fieldOfView** [comment]
Let's await the feedback from the beta in a couple of weeks

**Pull request merged**